synth_data_module/HCAI_inpatient_format.py

```diff
@@ -0,0 +1,155 @@
+import datetime as dt
+from synth_data_module import HCAIBase, get_procedure_list, get_diagnosis_list
+import pandas as pd
+import numpy as np
+import synth_data_module.mappings as mappings
+import time
+from io import StringIO
+
+class HCAIInpatientFormat(HCAIBase):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
Here (and in HCAI_PDD_format.py), you want to be more explicit about the args that you want to take. You can still have the caller pass in `**kwargs`,